tests/test_coupons.py

```diff
@@ -122,6 +123,17 @@ def test_delete(self):
             coupon.refresh_from_db()
             self.assertTrue(coupon.is_deleted)
 
+        with mock.patch("aa_stripe.models.StripeCoupon.delete") as mocked_delete:
+            # also test the overriden queryset's delete
+            coupon2 = self._create_coupon(coupon_id="CPON2")
+            coupon3 = self._create_coupon(coupon_id="CPON3")
+            coupons_qs = StripeCoupon.objects.filter(pk__in=[coupon2.pk, coupon3.pk])
+            self.assertEqual(coupons_qs.count(), 2)
+            delete_result = coupons_qs.delete()
+            self.assertEqual(mocked_delete.call_count, 2)
+            self.assertEqual(delete_result, (0, {"aa_stripe.StripeCoupon": 0}))
```

Comment:
This also looks as wrong value. I am not certain, but should it be :(2, {"aa_stripe.StripeCoupon": 2}))?
Anyway, there should be 0 in line 135.

Reply:
@poxip this still is invalid. https://github.com/django/django/blob/master/django/db/models/deletion.py#L318 - I clearly adressed it earlier. Not going to wait for it to be finished, I'll take care of it but this is again an example of missed comment..